code/modules/surgery/organs/organ_external.dm

```diff
@@ -312,7 +312,8 @@ This function completely restores a damaged organ to perfect condition.
 
 	// handle internal organs
 	for(var/obj/item/organ/internal/current_organ in internal_organs)
-		current_organ.rejuvenate()
+		if(!istype(current_organ,/obj/item/organ/internal/cyberimp))
```

Comment:
This doesn't actually address the source of the problem, here: 
https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/living/carbon/human/human.dm#L1314-L1316

This calls the `create_organs` of the human's current species, to regrow things like arms and missing vital organs; After all, it is difficult to call the `rejuvenate` of an organ which does not exist.

However: 
https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/living/carbon/human/species/species.dm#L149-L155

These lines are what obliterate present implants; This means you will lose all augment organs on a species change.